pkg/scheduler/api/node_info.go

```diff
@@ -34,6 +34,11 @@ func (o *AllocateFailError) Error() string {
 	return o.Reason
 }
 
+type NodeUsage struct {
```

Comment:
prefer to seperate this one into two PRs: shuffle action & rescheduling plugin.

Reply:
In order to make the implementation looks complete, how about separate the PR into 2 commits within the PR?

Reply:
> In order to make the implementation looks complete and work as a whole feature, how about separate the PR into 2 commits?

shuffle action is a common feature which is not related to rescheduling. TDM plugin also use this action :)

Reply:
> In order to make the implementation looks complete and work as a whole feature, how about separate the PR into 2 commits?

shuffle action is a common feature which is not related to rescheduling. TDM plugin also use this action :)

Reply:
Got it. That's reasonable. It's ready now.

Reply:
let's have a seperate PR for shuffle action, not a commit.